module/Application/src/Application/View/Helper/Acl.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Application\View\Helper;
+
+use Zend\View\Helper\AbstractHelper;
+
+
+class Acl extends AbstractHelper
+{
+
+    /**
+     * Acl
+     *
+     * @var \Zend\Permissions\Acl\Acl
+     */
+    protected $acl;
+
+    /**
+     * Get the current user's role.
+     *
+     * @return User|string
+     */
+    protected $role;
+
+    /**
+     * Check if a operation is allowed for the current role.
+     *
+     * @param string $operation Operation to be checked.
+     * @param string|ResourceInterface $resource Resource to be checked
+     *
+     * @return boolean
+     */
+    public function __invoke($operation, $resource)
```

Comment:
I believe this would become a bit confusing since our isAllowed implementation in the service has them in this order https://github.com/GEWIS/gewisweb/blob/master/module/Application/src/Application/Service/AbstractAclService.php#L49

Also I can't think of many cases in which the operation should be empty.